isis/CMakeLists.txt

```diff
@@ -425,6 +425,11 @@ EXECUTE_PROCESS(COMMAND cp -f ${CMAKE_SOURCE_DIR}/TestPreferences ${CMAKE_BINARY
 install(CODE "EXECUTE_PROCESS(COMMAND cp -f ${CMAKE_SOURCE_DIR}/src/base/objs/Preference/TestPreferences ${CMAKE_INSTALL_PREFIX}/)")
 install(CODE "EXECUTE_PROCESS(COMMAND cp -f ${CMAKE_SOURCE_DIR}/IsisPreferences ${CMAKE_INSTALL_PREFIX}/)")
 
+# Install the header files
+install(CODE "EXECUTE_PROCESS(COMMAND mkdir -p ${CMAKE_INSTALL_PREFIX}/include/isis3)")
+file(GLOB ISIS_INCLUDE_FILES ${CMAKE_BINARY_DIR}/inc/*)
+install(CODE "EXECUTE_PROCESS(COMMAND cp -rfv ${ISIS_INCLUDE_FILES} ${CMAKE_INSTALL_PREFIX}/include/isis3)")
```

Comment:
I am concerned about using the `-r` option on OSX. The version of `cp` that ships with OSX doesn't support it (using `-R` instead). We already require the gnu ports to run the tests, this would require them to build also.

Reply:
After testing, you do not need the `-r` on here. Once that is removed this is ready to go.